packages/jmix-front-generator/src/building-blocks/stages/answers/defaultGetAnswersFromOptions.ts

```diff
@@ -0,0 +1,25 @@
+import {ProjectModel} from "../../../common/model/cuba-model";
+import {YeomanGenerator} from "../../YeomanGenerator";
+import {CommonGenerationOptions} from "../../../common/cli-options";
+import {StudioTemplateProperty} from "../../../common/studio/studio-model";
+import { refineAnswers } from "./pieces/refineAnswers";
+import {throwError} from "../../../common/utils";
+
+export const defaultGetAnswersFromOptions = <O extends CommonGenerationOptions, A>(
+    projectModel: ProjectModel, gen: YeomanGenerator, options: O, questions: StudioTemplateProperty[]
+): A => {
+  if (options.answers == null) {
+    throwError(gen, 'Answers not found in options');
+  }
+
+  gen.log('Skipping prompts since answers are provided');
+  const rawAnswers = parseBase64Object(options.answers);
+  return refineAnswers(projectModel, questions, rawAnswers);
+}
+
+function parseBase64Object<O extends CommonGenerationOptions>(base64EncodedObject: string) {
```

Comment:
We have `base64encode` function in `util.ts`. Also there are several places in project when we parse base64. May be it make sense to move `parseBase64` in `util.ts` and refactor all places, where parsing used. I think it's better to extract this in separate ticket.